arm/Microsoft.MachineLearningServices/workspaces/.parameters/min.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "<<namePrefix>>-az-mls-min-001"
+        },
+        "sku": {
+            "value": "Basic"
+        },
+        "associatedStorageAccountResourceId": {
+            "value": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Storage/storageAccounts/adp<<namePrefix>>azsax001"
+        },
+        "associatedKeyVaultResourceId": {
+            "value": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.KeyVault/vaults/adp-<<namePrefix>>-az-kv-x-001"
+        },
+        "associatedApplicationInsightsResourceId": {
+            "value": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Insights/components/adp-<<namePrefix>>-az-appi-x-001"
+        },
+        "systemAssignedIdentity": {
```

Comment:
I'd suggest to remove it since it's not a required parameter

Reply:
I think I had to add it as it failed without. Will retest

Reply:
In that case would the parameter be required/conditional?

Reply:
![image](https://user-images.githubusercontent.com/5365358/166680904-7b2e0626-3397-40e3-a461-4c6b82830ede.png)

Must be there in some capacity. Will update

Reply:
@eriqua, it's true, either you need set `systemAssignedIdentity` to true, or set both `userAssignedIdentities` & `primaryUserAssignedIdentity`. Tested both succesfully